Packs/Hudsonrock/Integrations/Hudsonrock/Hudsonrock.py

```diff
@@ -0,0 +1,113 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def __init__(self, base_url: str, headers: dict, proxy: bool = False, verify: bool = False):
+        super().__init__(base_url=base_url, headers=headers, proxy=proxy, verify=verify)
+
+    def query(self, suffix: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=suffix
+        )
+
+def test_module(client: Client, query: str) -> str:
+    result = client.query(query)
+    if result:
+        return 'ok'
+    else:
+        return 'Test failed: ' + str(result)
+
+def create_indicator_output(results: Dict[str, Any], indicator: str, indicatortype: str, reliability: str) -> CommandResults:
+    if indicatortype == 'ip':
+        indicator_type = DBotScoreType.IP
+    else:
+        indicator_type = DBotScoreType.EMAIL
+
+    if 'is associated' in results['message']:
+        dbot_score_object = Common.DBotScore(indicator=indicator, indicator_type=indicator_type,
+                                             integration_name='Hudsonrock', score=3, reliability=reliability)
+    else:
+        dbot_score_object = Common.DBotScore(indicator=indicator, indicator_type=indicator_type,
+                                             integration_name='Hudsonrock', score=0, reliability=reliability)
+    indicator_to_return: Union[Common.IP, Common.EMAIL]
+    if indicatortype == 'ip':
+        indicator_to_return = Common.IP(dbot_score=dbot_score_object, ip=indicator)
+        outputs_prefix = 'IP'
+    else:
+        indicator_to_return = Common.EMAIL(dbot_score=dbot_score_object, address=indicator)
+        outputs_prefix = 'Email'
+
+    human_readable = tableToMarkdown('Hudsonrock results', results)
+
+    return CommandResults(
+        outputs_prefix=outputs_prefix,
+        outputs_key_field=indicator,
+        outputs=results,
+        indicator=indicator_to_return,
+        readable_output=human_readable
+    )
+
+
+
+def create_output(results: Dict[str, Any], endpoint: str, keyfield: str = '') -> CommandResults:
+    human_readable = tableToMarkdown('Hudsonrock results', results)
+    return CommandResults(
+        outputs_prefix=f'Hudsonrock.{endpoint}',
+        outputs_key_field=keyfield,
+        outputs=results,
+        readable_output=human_readable
+    )
+
+
+
+def main():
+    base_url = 'https://cavalier.hudsonrock.com/api/json/v2/osint-tools/'
```

Comment:
Not a must, but i suggest to make it a parameter(server/service URL), put this as the default value: "https://cavalier.hudsonrock.com", and add the "api/json/v2/osint-tools/" in the code. this way the user have control if there is redirection or special service address.